Dockerfile

```diff
@@ -11,13 +11,17 @@ WORKDIR /src/
 ADD go.mod .
 ADD go.sum .
 ADD vendor vendor
-RUN go mod download
+
+# Do not download neither verify vendored folder, yip should be bumped to a recent version as
+# some of its dependencies are no longer available upstream. Relay entirely on vendored folder
+# RUN go mod download
+
```

Comment:
This is to relay entirely to the vendored folder, which I think is fine for a maintenance branch. Other wise we will see failure in CI as it happened in this [run](https://github.com/rancher/elemental-toolkit/actions/runs/9989529687?pr=2145)